app/controllers/searches.js

```diff
@@ -0,0 +1,28 @@
+import Controller from '@ember/controller';
+import EmberObject, { computed } from '@ember/object';
+import { oneWay } from '@ember/object/computed';
+import { empty } from '@ember/object/computed';
+import { alias } from '@ember/object/computed';
+
+
+export default Controller.extend({
+ queryParams: ['searchkey'],
+ searchkey:null,
+ keywordSearches: null,
+ resultLength: alias('model.length'),
+ searchKeyChange: alias('searchkey'),
+ keywordSearches: oneWay('searchKeyChange'),
```

Comment:
why are you doing oneway binding? if you are concerned about the query to the back end, you can always trigger the query with an action. should for the most part keep two way binding.

Reply:
Back button is fixed. one way binding is removed